vscode/src/debugger/activate.ts

```diff
@@ -4,21 +4,28 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 
 import * as vscode from "vscode";
-import { ICompiler, getCompilerWorker } from "qsharp";
+import { IDebugServiceWorker, getDebugServiceWorker } from "qsharp";
 import { FileAccessor, qsharpExtensionId } from "../common";
 import { QscDebugSession } from "./session";
 
-let compiler: ICompiler;
+let debugServiceWorkerFactory: () => IDebugServiceWorker;
 
 export async function activateDebugger(
   context: vscode.ExtensionContext
 ): Promise<void> {
-  const workerScriptPath = vscode.Uri.joinPath(
+  const compilerWorkerScriptPath = vscode.Uri.joinPath(
```

Comment:
Why do you need the compilerWorkerScriptPath now? (If you remove line 4 does that help catch these issue? Or too noisy?)

Reply:
Missed during refactoring when I pulled out the compiler.